lib/index.js

```diff
@@ -47,10 +47,10 @@ function NwBuilder(options) {
         argv: process.argv.slice(2)
     };
     // Intercept the platforms and check for the legacy platforms of 'osx' and 'win' and
-    // replace with 'osx32', 'osx64', and 'win32', 'win64' respectively.
+    // replace with 'osx64', and 'win32', 'win64' respectively.
     if(typeof options.platforms != 'undefined'){
         if(options.platforms.indexOf('osx') >= 0){
-            options.platforms.splice(options.platforms.indexOf('osx'), 1, 'osx32', 'osx64');
+            options.platforms.splice(options.platforms.indexOf('osx'), 1, 'osx64');
```

Comment:
That's wholly up to you. I would agree that it's a breaking change, but in that same respect all of it is: `osx32` used to be built by default, and now it won't be. I'd think that the opt-in to all OSX platforms with `--platforms osx` is more explicit than using the default, and more likely to cause problems with being changed (which I imagine is what you're thinking).

So, I'll remove this bit presently.